packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -105,6 +106,7 @@ class _MyHomePageState extends State<MyHomePage> {
     // than having to individually change instances of widgets.
     return Scaffold(
       appBar: AppBar(
+        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
```

Comment:
should this just be the default? it seems weird that we're so unhappy with our defaults that we're overriding them in what should be the app with the biggest motivation to use all the defaults. cc @HansMuller

Reply:
We're planning to migrate to useMaterial3:true by default in stages. In addition to publishing a migration guide and making sure that developers know what's coming, in this stage we'll make the counter app use material 3. In the following stage we'll make it true by default.

Reply:
The default AppBar background color for Material 3 matches the body's background color (white). That's why the new version of the counter app painted a colorful star, so that changes to the theme's color scheme had a more dramatic impact. If the star has to be removed, then we wanted something that would highlight color scheme changes, and since there's already an app bar... Personally, I think the star added a little visual interest, along with highlighting color scheme changes.